lib/src/transformer/impl_generation.dart

```diff
@@ -617,6 +608,24 @@ class ImplGenerator {
         sourceFile.location(typedMap.node.leftBracket.end),
         staticVariablesImpl
     );
+
+    final name = typedMap.node.name.name;
+    final metaClassName = '$generatedPrefix${name}Meta';
+    final metaStructName = type == AccessorType.props
+        ? 'PropsMeta'
+        : 'StateMeta';
+    final metaInstanceName = '${publicGeneratedPrefix}metaFor$name'; // FIXME validate boilerplate to avoid typos
```

Comment:
This note was from the original implementation, @greglittlefield-wf what validation did you have in mind?

Reply:
I believe this is referring to validating that the `meta` variable is referencing the correct generated variable.

For example, for `FooProps`,
```dart
class FooProps {
  static const PropsMeta meta = $metaForFooProps;
}
```
making sure they don't have `= $metaForBarProps` or something, which could easily happen as a form of copy-paste error.

Reply:
validation implemented here: https://github.com/Workiva/over_react/pull/206/files#diff-92afc98c256ac39698eadfad4f3a3ce4R209